install.md

```diff
@@ -204,6 +204,10 @@ The build steps on Debian are otherwise the same as Ubuntu, above.
 
 ## Configuration files
 
+The following configuration files are required in order for Buildah to run appropriately.  The
+majority of these files are contained in the `containers-common` package, please verify that
```

Comment:
Can you state that this is commonly packaged in containers-common.  No need to force all distros to package that way.

Reply:
hmm, didn't get e-mail on this.  I've add commonly and have removed ', please verify...' as it tells them to check for that kit.